app/components/windows/ChildWindow.vue.ts

```diff
@@ -15,11 +15,13 @@ export default class ChildWindow extends Vue {
 
   components: { name: string; isShown: boolean; }[] = [];
 
+  private refreshingTimeout = 0;
+
   mounted() {
-    this.onWindowUpdatedHandler();
-    this.windowsService.windowUpdated.subscribe(params => {
-      if (params.windowId !== 'child') return;
-      this.onWindowUpdatedHandler();
+    this.onWindowUpdatedHandler(this.options);
```

Comment:
is this the only place we call `onWindowUpdatedHandler`? otherwise we need to include options in our other calls

Reply:
`onWindowUpdatedHandler` is private. So yes it can be called only from current file